docs/first-order/classify_waveforms.ipynb

```diff
@@ -0,0 +1,2327 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # function that returns a list of channels ids that are valid within electrodes table
</pre><p>can remove 'function that'. You want comments to be as brief as possible, but not more brief.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>